LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2016-2020, QIIME 2 development team.
```

Comment:
Drive-by-comment: the copyright should just be for 2020, since that is the year this plugin was created in.